g2p_registry_rest_api/services/process_individual_mixin.py

```diff
@@ -20,7 +20,7 @@ def _process_individual(self, individual):
             "given_name": individual.given_name,
             "family_name": individual.family_name,
             "addl_name": individual.addl_name,
-            "gender": individual.gender.capitalize() if individual.gender else False,
+            "gender": individual.gender if individual.gender else False,
```

Comment:
This depends on what is the API response we want to give. For example; If we want to give the internal representation for "Male" (that is "male"), then this is ok. If we want to give actual gender as being shown on UI (?) then this needs to change so that this value is picked from the selection fields.